cli/commands/configure.py

```diff
@@ -28,8 +28,6 @@ def configure(ctx: CliContext,
     profile = ctx.confman.conf.active_profile
 
     token = token or profile.token
-    if token is None:
```

Comment:
There is no problem that the user will provide token in other commands (`execute` command for example).

In `configure` command, the user tries to create / edit a profile. 
If token is None after `token = token or profile.token`, it means the user tries to create a new profile (If he would want to edit an existing profile, we would have found his token with `profile.token`).

Therefore the user must provide a toke.
If we can't find any token, we cant create a new profile.

Reply:
There might be a case for it though IMO this is an edge case.
The typical case here is to create multiple profiles each with its own token, then call each command with the proper profile.
Since we removed token generation from the CLI there is no dynamic token usage (The user will play around with the same tokens over and over).